### DISCUSSION TITLE
Minimum/zero/maximum in hv display

### DISCUSSION THREAD

**Discussion by hogass83** [agronick/aa-torque#79] [Ideas]
When using gauges that can go negative is there a way to make the high visibility gauge start at zero instead of the minimum number? Example being incline. It starts at -90° rather than 0°. Thanks for any input, youve truly done awesome work with this. I love this app!!!!

**Comment by agronick** [comment]
Just set the min value in the settings. If you mean the first reading, it used to be 0 but now it shows the min value so nothing is displayed on the gauge.

**Reply by hogass83** [reply]
But if minimum value is say -90° so 0° is 12 o clock it shades -90 to 0 when reading is 0.

**Reply by agronick** [reply]
I forked the gauge library. I tweaked it to make this work. Going to test it today.

**Reply by hogass83** [reply]
Awesome!!! I'll be donating asap! You've done an exceptional job on this project!

**Reply by agronick** [reply]
Thanks if you check out the prerelease version it is working.

**Reply by hogass83** [reply]
PERFECTION! YOU'RE AWESOME!!!!!!!
